search/src/main/scala/weco/api/search/elasticsearch/WorksMultiMatcher.scala

```diff
@@ -83,25 +91,31 @@ case object WorksMultiMatcher {
             MultiMatchQuery(
               q,
               queryName = Some("title and contributor exact spellings"),
-              fields = fieldsWithBoost(boost = 100, fields = titleFields),
+              fields = fieldsWithBoost(
+                boost = 100,
+                fields = titleAndContributorFields.flatMap(
+                  field =>
+                    Seq(
+                      field,
+                      s"$field.english",
+                      s"$field.shingles"
+                    )
+                )
+              ),
               `type` = Some(BEST_FIELDS),
               operator = Some(AND)
             ),
             MultiMatchQuery(
               q,
               queryName = Some("non-english titles and contributors"),
-              fields = List(
-                "arabic",
-                "bengali",
-                "french",
-                "german",
-                "hindi",
-                "italian"
-              ).map(
-                language =>
-                  FieldWithOptionalBoost(
-                    s"search.titlesAndContributors.$language",
-                    None
+              fields = titleAndContributorFields.flatMap(
+                field =>
+                  nonEnglishLanguages.map(
+                    language =>
+                      FieldWithOptionalBoost(
```

Comment:
Given the frequency with which `FieldWithOptionalBoost(???, None)` is used, I wonder if it would be cleaner to have a `Field` object that does not take a boost argument.

Reply:
Good suggestion, I've added `FieldWithoutBoost`.